src/index.ts

```diff
@@ -345,6 +349,54 @@ class Spanner extends GrpcService {
       [CLOUD_RESOURCE_HEADER]: this.projectFormattedName_,
     };
     this.directedReadOptions = directedReadOptions;
+    this.emulatorHost = emulatorHost;
+  }
+
+  // Helper for instance admin client API calls
```

Comment:
Methods that are intended for public use should be documented using jsdoc, so using comments in the form `/** method documented here. */

In this case, the documentation should also include an instruction that the returned client is managed by the Spanner client, and that it should not be closed by the user, but will be closed by the client library when `Spanner.close()` is called.